src/functions-and-arrays.js

```diff
@@ -1,73 +1,450 @@
 // Iteration #1: Find the maximum
-
+const maxOfTwoNumbers = (num1, num2) => {
+  if (num1 > num2) return num1;
```

Comment:
You can instead write:
if(num1 >= num2) return num1

This way you will not need your: 
else return num1